cranecloud/docs/guides/deployHtmlApp.md

```diff
@@ -0,0 +1,89 @@
+Getting an application running on Crane Cloud is essentially working out how to package it as a deployable image. Once packaged it can be deployed to the Crane Cloud platform.
+
+In this guide, we will learn how to containerize a sample HTML Application using Docker and then deploy to Crane Cloud.
+
+In this guide, we will learn how to containerize a sample HTML Application using Docker and then deploy to Crane Cloud. See [demo here](https://my-html-app-578060fa-3901-4464-8f3d.cranecloud.io/)
```

Comment:
@rhodinemma repetition of the preceding line 3 above.

Reply:
my bad! let me resolve that. thanks for noticing.